tests/readConcern/readconcern-bsonserialize-001.phpt

```diff
@@ -11,6 +11,7 @@ $tests = [
     new MongoDB\Driver\ReadConcern(MongoDB\Driver\ReadConcern::LOCAL),
     new MongoDB\Driver\ReadConcern(MongoDB\Driver\ReadConcern::MAJORITY),
     new MongoDB\Driver\ReadConcern(MongoDB\Driver\ReadConcern::AVAILABLE),
+    new MongoDB\Driver\ReadConcern(MongoDB\Driver\ReadConcern::SNAPSHOT),
```

Comment:
It seems odd to me that we exhaustively test every ReadConcern constant in all of these tests (as opposed to just `readconcern-constants.phpt`), but new constants are not introduced frequently enough to make me care about stripping these down.

Reply:
https://jira.mongodb.org/browse/PHPC-1929